documentation/modules/proc-changing-a-topic.adoc

```diff
@@ -5,7 +5,17 @@
 [id='changing-a-topic-{context}']
 = Changing a topic
 
-This procedure describes how to change the configuration of an existing Kafka topic by using a `KafkaTopic` Kubernetes resource.
+This procedure describes how to change the configuration of an existing Kafka topic by using a `KafkaTopic` resource.
+
+.Before you start
+
+It is important that you consider the following before making your changes:
+
+* Kafka does _not_ support making the following changes through the `KafkaTopic` resource:
+** Changing topic names using `spec.topicName`
+** Decreasing partition size using `spec.partitions`
+** Changing the number of replicas using `spec.replicas`
```

Comment:
This is as much a Strimzi limitation as a Kafka one, so maybe `*` rather than `**`.